build.gradle.kts

```
@@ -2,10 +2,38 @@ plugins {
     //trick: for the same plugin versions in all sub-modules
     id(Plugins.ANDROID_APPLICATION).version(Versions.GRADLE).apply(false)
     id(Plugins.ANDROID_LIBRARY).version(Versions.GRADLE).apply(false)
+    id(Plugins.DETEKT).version(Versions.DETEKT)
```

We need to set a global version of Kover here, same to Detekt

```suggestion
    id(Plugins.DETEKT).version(Versions.DETEKT)
    id(Plugins.KOVER).version(Versions.KOVER)
```

---

I see. 👍🏻  I just follow the instruction from the Kover document 😂